packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -45,9 +45,6 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 	private readonly editManager: EditManager<TEditor, TChange, ChangeFamily<TEditor, TChange>>;
 	private readonly summarizables: readonly Summarizable[];
 
-	/** Iff false, calls to `submitOp` will have no effect */
-	private submitOps = true;
```

Comment:
@microsoft/fluid-cr-tree i'd discussed this change with @noencke in the past, but would still like confirmation this looks correct.